content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,188 @@
-balance or to recharge your account, go to :menuselection:`Settings --> Odoo IAP --> View my
-Services`.
+IAP services do **not** need to be configured or set up before use. Odoo users can simply click on
+the service in the app, to activate it. However, each service requires its own prepaid credits, and
+when they run out, users **must** :ref:`buy more <iap/buying_credits>` in order to keep using it.
 
-.. image:: in_app_purchase/image1.png
-   :align: center
+.. note::
+   Users with an Enterprise version of Odoo Online get free credits to test IAP features.
 
-.. tip::
-   If you are on Odoo Online and have the Enterprise version, you benefit from free credits to test our
-   IAP features.
+.. _in_app_purchase/portal:
 
-IAP accounts
+IAP services
 ============
 
-Credits to use IAP services are stored on IAP accounts, which are specific to each service.
-By default, IAP accounts are common to all companies, but can be restricted to specific
-ones. Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Technical
-Settings --> IAP Account`.
+IAP services are provided by Odoo, as well as third-parties, and have a wide range of use cases.
 
-.. image:: in_app_purchase/image2.png
-   :align: center
+The following IAP services are provided by Odoo:
 
-.. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+- :guilabel:`Documents Digitization`: digitizes scanned or PDF vendor bills, expenses, and resumes
+  with OCR and AI.
+- :guilabel:`Partner Autocomplete`: automatically populates contact records with corporate data.
+- :guilabel:`SMS`: sends SMS text messages to contacts directly from the database.
+- :guilabel:`Lead Generation`: generates leads based on a set of criteria and converts web visitors
+  into quality leads and opportunities.
+- :guilabel:`Snailmail`: sends customer invoices and follow-up reports by post, worldwide.
 
-IAP Portal
-==========
+For more information on every service currently available, visit the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_.
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+Use IAP service
+---------------
 
-.. image:: in_app_purchase/image3.png
-   :align: center
+IAP services are automatically integrated with Odoo, and do **not** require users to configure any
+settings. To use a service, simply interact with it wherever it appears in the database.
 
-Get notified when credits are low
-=================================
+.. example::
+   The following flow focuses on the *SMS* IAP service being used from a contact's record. This can
+   be done by clicking the :guilabel:`📱 (phone) SMS` icon within the database.
```

Comment:
```suggestion
   The following flow focuses on the *SMS* IAP service being used from a contact's record.
   
   This can be done by clicking the :guilabel:`📱 (phone) SMS` icon within the database.
```